package.json

```diff
@@ -50,7 +50,7 @@
     "node": ">6.0.0"
   },
   "dependencies": {
-    "@webcomponents/custom-elements": "^1.0.0-alpha.3",
+    "@webcomponents/custom-elements": "github:webcomponents/custom-elements#v1.0.0-alpha.4",
```

Comment:
For now, I'm pulling directly from Github, since "alpha 4" has not been published to NPM. I made an issue requesting that it be published there, which can be found here:

https://github.com/webcomponents/custom-elements/issues/68